packages/node_modules/samples/browser-meetings/index.html

```diff
@@ -237,38 +237,21 @@ <h2>Meeting Controls</h2>
           <legend>Participants</legend>
           <div>
             <input id="add-participant-query" name="participantQuery" placeholder="Email Address" value="" type="email" />
-            <button id="add-participant" type="button">Add</button>
+            <button id="add-participant" onclick="inviteMember(this)" type="button">Add</button>
           </div>
-          <br />
-          <div>
-            <button id="participant-a-admit" type="button" value="[[PARTICIPANT ID]]">Admit</button>
-            <button id="participant-a-remove" type="button" value="[[PARTICIPANT ID]]">Remove</button>
-            <button id="participant-a-mute" type="button" value="[[PARTICIPANT ID]]">Mute</button>
-            <button id="participant-a-transfer-ownership" type="button" value="[[PARTICIPANT ID]]">Transfer Ownership To</button>
-            <label id="participant-a-id">[[PARTICIPANT ID]]</label>
-            <label id="participant-a-status">[[PARTICIPANT STATUS]]</label>
-            <label id="participant-a-name">[[PARTICIPANT NAME]]</label>
-          </div>
-          <div>
-            <button id="participant-b-admit" type="button" value="[[PARTICIPANT ID]]">Admit</button>
-            <button id="participant-b-remove" type="button" value="[[PARTICIPANT ID]]">Remove</button>
-            <button id="participant-b-mute" type="button" value="[[PARTICIPANT ID]]">Mute</button>
-            <button id="participant-b-transfer-ownership" type="button" value="[[PARTICIPANT ID]]">Transfer Ownership To</button>
-            <label id="participant-b-id">[[PARTICIPANT ID]]</label>
-            <label id="participant-b-status">[[PARTICIPANT STATUS]]</label>
-            <label id="participant-b-name">[[PARTICIPANT NAME]]</label>
-          </div>
-          <div>
-            <button id="participant-c-admit" type="button" value="[[PARTICIPANT ID]]">Admit</button>
-            <button id="participant-c-remove" type="button" value="[[PARTICIPANT ID]]">Remove</button>
-            <button id="participant-c-mute" type="button" value="[[PARTICIPANT ID]]">Mute</button>
-            <button id="participant-c-transfer-ownership" type="button" value="[[PARTICIPANT ID]]">Transfer Ownership To</button>
-            <label id="participant-c-id">[[PARTICIPANT ID]]</label>
-            <label id="participant-c-status">[[PARTICIPANT STATUS]]</label>
-            <label id="participant-c-name">[[PARTICIPANT NAME]]</label>
+          <p>New participants will be added below</p>
+          <div class="particpantList">
```

Comment:
This list should display a label much like the `current meetings list` section when empty.
Additionally, see the `generateMeetingsListItem()` method for an example function on how one could add elements to the page using the provided data.

Please modify this HTML, as well as the associated functions with this div, to allow for this list to be generated on button press with the functional buttons on each list item.